Resources/Prototypes/_CD/Voice/lay_emotes.yml

```diff
@@ -0,0 +1,37 @@
+- type: emote
+  id: LayDown
+  chatMessages: [lays down]
+  chatTriggers:
```

Comment:
this weirdly doesn't allow any other emote *except* voluntarily lying down... not sure what the reasoning was on CD but what I want to fall over? be pushed over? :O